File: frontend/src/model/UsersEva.ts
import Evaluation from "./Evaluation";

type UserEva={
    mail : string;
    accountName : string;
    manageFirstName : string;
    manageLastName : string;
    companyName : string;
    companyStreet : string;
    companyZip : string;
    companyLocation : string;
    employee1: string;
    employee2: string;
    employee3: string;
    employee4: string;
    employee5: string;
    medicalCareName : string;
    medicalCareStreet : string;
    medicalCareZip : string;
    medicalCareLocation : string;
    evaName: string;
    forWhom:string;
    forWhomStreet:string;
    forWhomZip:string;
    forWhomLocation: string;
    evaCatalog:Array<Evaluation>;
}

export default UserEva;

File: backend/src/main/java/de/neuefische/backend/model/UsersEva.java
package de.neuefische.backend.model;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "usersEva")
public class UsersEva {

    @Id
    private String id;

    private String accountName;
    private String mail;
    private String manageFirstName;
    private String manageLastName;
    private String companyName;
    private String companyStreet;
    private String companyZip;
    private String companyLocation;
    private String employee1;
    private String employee2;
    private String employee3;
    private String employee4;
    private String employee5;
    private String medicalCareName;
    private String medicalCareStreet;
    private String medicalCareZip;
    private String medicalCareLocation;
    private String evaName;
    private String forWhom;
    private String forWhomStreet;
    private String forWhomZip;
    private String forWhomLocation;
    private List<Evaluation> evaCatalog;

}
